### ISSUE TITLE
Feature request: link from clue to grid

### ISSUE THREAD

**Issue by elle-trudgett** [crosshare-org/crosshare#434]
Would be nice to click the word or the location (e.g. 4D) in the Clues tab of the constructor to switch to the grid and highlight / select the word.

**Comment by mdirolf** [comment]
I like this idea. I have been mulling over what the best way would be to move cluing onto the grid view - it'd be nice to be able to view/add/edit clues w/o leaving the grid page. That might obviate this issue or could be done in addition, depending on how it ends up looking.

**Comment by elle-trudgett** [comment]
Is the main hangup keeping the interface responsive for mobile? (Or is that kind of consigned to larger-format devices already, anyway?) I'd imagine you could create minis on mobile with relative ease, at least.

I think having the right pane switchable between word completion and clueing would be good, because once I'm basically done with a crossword I hardly use the word completion pane (and if I'm honest, I don't use it that often to begin with) I'm switching back and forth between clues and the grid (to ensure the difficulty of clues isn't too clumped up) and also to change bad auto-fills I hadn't originally noticed.

Perhaps that could work, with the ability to resize the panes (allow more space for clues while viewing a minified version of the grid on the left.) The cluing interface would need to update in real time as the fills were changed, but that is probably not a big issue in terms of UX because they won't be in the middle of editing clues if they're changing the fills.

**Comment by mdirolf** [comment]
Yeah coming up with a design that works for mobile too is definitely a big part of what needs to happen. Otherwise I am envisioning something similar to what you're describing.

**Comment by mdirolf** [comment]
Related #351
